action/src/plugins/pluginRunner.ts

```diff
@@ -6,12 +6,19 @@ import { runStaticHostingPlugin } from './staticHosting';
 
 export const runPlugins = async (
   stoatConfig: StoatConfigSchema,
+  stepsSucceeded: boolean,
   githubActionRun: GithubActionRun,
```

Comment:
I think that adding an extra parameter in the signature is unnecessary. Ideally we don't want to add an extra parameter each time there is a new feature.

The `githubActionRun` has a `ghJob` field that is the workflow job object. The step success information is included in that object. My first proposal is to move the calculation of the `stepsSucceeded` from the `app.ts` to the static hosting plugin function based on the `ghJob` field.

If the first proposal is overkill, or you prefer to log the job status in the main `app.ts`, my second proposal is that the `stepSucceeded` variable can be added as a new field under `GithubActionRun`. This approach makes sense in that the success status is a metadata for the action run. Also this avoid adding an extra parameter in the signature.

Reply:
I'll do the latter.

Reply:
We will probably want to use it for non-static hosting plugins in the future.